tests/src/test/java/stepDefinitions/backendstepdefinitions/GenerateEfilingUrl.java

```diff
@@ -99,22 +102,43 @@ public static void user_calls_incorrect_with_POST_http_request(String resource)
         APIResources resourceInvalid = APIResources.valueOf(resource);
 
         request = given().spec(TestUtil.requestSpecification()).body(payloadData.generateUrlFinalPayload());
-        response = request.when().post(resourceInvalid.getResource() + "s").then().extract().response();
+        response = request.when().post(resourceInvalid.getResource()).then().extract().response();
     }
 
     @When("status is {int} and content type is verified")
     public static void status_is_and_content_type_is_verified(Integer int1) {
```

Comment:
```suggestion
    public static void statusIsAndContentTypeIsVerified(Integer int1) {
```
use java naming convention for methods names

Reply:
Sure. The current implementation was based on the Cucumber stepdefinition naming conventions. I'll switch to Java convetion. Thanks!

Reply:
is cucumber java specific ?

Reply:
No its not java specific. Can be used with other languages as well. It just generates the method names like that during the dry run. Updated the method names in java convention. Will continue that in this project. Thank you!